azure/terraform/stacks/vault-injector/external_dns.tf

```diff
@@ -3,8 +3,11 @@ resource "vault_kv_secret_v2" "externaldns_keyid" {
   name                       = "externaldns/service_account"
   data_json                  = jsonencode(
   {
-    key_id = data.terraform_remote_state.acm_general.outputs.externaldns_sp_keyid,
-    key_secret = data.terraform_remote_state.acm_general.outputs.externaldns_sp_password
+    aadClientId = data.terraform_remote_state.acm_general.outputs.externaldns_sp_appid,
+    aadClientSecret = data.terraform_remote_state.acm_general.outputs.externaldns_sp_password
+    resourceGroup = "acm-general"
```

Comment:
should we get this from a var / local?

Reply:
Yeah. You're right. I've updated it to pull that info in [cc1430b](https://github.com/acm-uic/IaC/pull/58/commits/cc1430b71159470b52ebd2b5d3cbc644eacdde07).